sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -277,9 +288,13 @@ def recognize_linked_entities(  # type: ignore
         # type: (...) -> List[Union[RecognizeLinkedEntitiesResult, DocumentError]]
         """Recognize linked entities from a well-known knowledge base for a batch of documents.
 
-        Returns a list of recognized entities with links to a
-        well-known knowledge base. See https://aka.ms/talangs for
-        supported languages in Text Analytics API.
+        Identifies and disambiguates the identity of each entity found in text (for example,
+        determining whether an occurrence of the word Mars refers to the planet, or to the
+        Roman god of war). Returns a list of recognized entities with hyperlinks to a
```

Comment:
do we still keep hyperlink?

Reply:
Good catch, I'll change to URLs

Reply:
There are other places use the word in the PR